master/usage/configuration/node.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Configuring a node
```

Comment:
I'm wondering if this doc is really about "Configuring a node". From a user perspective, this is only about configuring BGP aspects of the node, i.e.  BGP IP, AS, and subnet. There's other node configuration (backend, etcd certs, yadda) and this doesn't touch on how you'd configure those things. 

It's ok we don't talk about it here but we should adjust the title, and consider linking to the node configuration reference as well